app.rb

```diff
@@ -16,7 +16,6 @@ def initialize
 
   def list_books
     puts 'No books added!' if @books.empty?
-
     @books.each { |book| puts "Title: #{book.title}, author: #{book.author}" }
     puts ''
   end
```

Comment:
- [ ] @SSEKPIUS and @Tamunokuro kindly ensure all requests for input and input validation should be placed in the main.rb file and/or classes/methods whose responsibilities are to request or validate input

Reply:
- [ ] @team also ensure your methods are following `Single responsibility rule` ,eg the create person method should have a statement to create student and teacher, the create student and create teacher method should be called in create person method to pass specialized parameter  unique to teachers and students